src/main/java/com/thebois/views/IActorView.java

```diff
@@ -0,0 +1,17 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+
+/**
+ * Represents a view that does not draw itself, but is rendered using actors instead.
+ */
+public interface IActorView {
+
+    /**
+     * Gets the root element of the view.
+     *
+     * @return The root element containing all other parts of the view.
+     */
+    Actor getRoot();
```

Comment:
Seems like a code smell? Generally a child should not know about its parent, maybe I am misunderstanding the implementation/use-case

Reply:
It's not a child knowing about its parent, it's more a parent knowing about its children. 

The `IActorView` interface is implemented by `RoleView`, which is actually just made up out of widgets from LibGDX. These widgets are all placed in a *root element* in `RoleView`, and in order to render `RoleView`, `GameScreen` needs to add this root element to the `Stage`. So it's actually the parent, `GameScreen`, getting its child through the `getRoot()` method.

Maybe the name is a bit confusing, do you have any ideas for different names?

Reply:
Maybe `getWidgetOwner` or somthing similar?

Reply:
in that case it's more of a widget container, it doesn't really own them, just positions them in a hierarchy with the top node being the "root" node. Hence the name `getRoot`. Would `getWidgetContainer` be clearer?